examples/bootstrapping-railing/railing.go

```diff
@@ -3,11 +3,15 @@ package main
 import (
 	"context"
 	"fmt"
-	"github.com/libp2p/go-libp2p"
-	"github.com/libp2p/go-libp2p-peerstore"
-	"github.com/ipfs/go-ipfs-addr"
+	
+	libp2p "github.com/libp2p/go-libp2p"
+	peerstore "github.com/libp2p/go-libp2p-peerstore"
+	ipfsaddr "github.com/ipfs/go-ipfs-addr"
 )
 
+//The bootstrapping nodes are just long running nodes with a static IP address.
+//That means you can easily have your own bootstrapping nodes. Everything you need is
+//a server with a static IP address.
```

Comment:
General go style is to have a space between // and comment.

Reply:
@Kubuxu I added it now. Do you know what `go fmt` didn't fix it?